source/features/latest-tag-button.tsx

```diff
@@ -111,21 +107,29 @@ async function init(): Promise<false | void> {
 		link.append(' ', <span className="css-truncate-target">{latestTag}</span>);
 	}
 
-	if (currentBranch === latestTag || isUpToDate) {
+	if (currentBranch === latestTag || aheadBy === 0) {
 		link.setAttribute('aria-label', 'You’re on the latest release');
 		link.classList.add('disabled', 'tooltipped', 'tooltipped-ne');
 		return;
 	}
 
 	const defaultBranch = await getDefaultBranch();
 	if (currentBranch === defaultBranch) {
-		const aheadBy = await getAheadByCount(latestTag);
-		if (!aheadBy) {
-			return;
+		link.append(<sup> +{aheadBy}</sup>);
+		link.setAttribute('aria-label', aheadBy ? `${defaultBranch} is ${pluralize(aheadBy, '1 commit', '$$ commits')} ahead of the latest release` : `The HEAD of ${defaultBranch} isn’t tagged`);
+
+		if (pageDetect.isRepoRoot()) {
+			const compareLink = (
+				<a
+					className="btn btn-sm btn-outline tooltipped tooltipped-ne"
+					href={`/${getRepoURL()}/compare/${latestTag}...${defaultBranch}`}
+					aria-label={`Compare ${latestTag}...${defaultBranch}`}
+				>
+					<DiffIcon/>
+				</a>
+			);
+			groupButtons([link, compareLink]);
```

Comment:
Can you move this to a new PR?

Reply:
Yes

Reply:
Done want me to redo the first post screen shots?